src/services/index.ts

```diff
@@ -1,10 +1,44 @@
 export * from './createParam'
 export * from './createRouter'
-export * from './path'
-export * from './query'
+export { withParams } from './withParams'
 export type { RouterRoute } from './createRouterRoute'
 export { withDefault } from './withDefault'
 export { createRouterPlugin } from './createRouterPlugin'
 export { unionOf } from './unionOf'
 export { arrayOf } from './arrayOf'
 export { tupleOf } from './tupleOf'
+
+// backwards compatible withParams
+import { withParams, WithParams, toWithParams } from './withParams'
+/**
+ * @deprecated use `toWithParams` instead
+ */
+export const toPath = toWithParams
```

Comment:
These `toPath` and other "to" utilities aren't part of our public api are they? Just the `path`, 'query`, etc utilities. So I think we can just delete the "to" utilities yeah?

Reply:
agreed. will remove